### DISCUSSION TITLE
Typing Error

### DISCUSSION THREAD

**Discussion by ZeroofSpade** [aclap-dev/video-downloadhelper#968] [Help and Q&A]
### Your question

I didn't receive the email regarding my premium license because I made a typing error and now I can't use the download helper. How do I resolve this?

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: pt-PT
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"locked":true,"key":"04e47f9193b045cd"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 6,
    "successfull_dl": 3
}

**Comment by jcfrog** [comment]
send me an email at licenses@downloadhelper.net with the spelling mistake you made
